src/tests/api/createUser.spec.js

```diff
@@ -0,0 +1,20 @@
+import { test } from "@playwright/test";
+import axios from "axios";
+import { expect } from "chai";
+
+const BASE_URL = "https://reqres.in/api";
+
+test.describe("Reqres Get API Tests", () => {
+  test("Getting single user by ID", async () => {
```

Comment:
bad test description, write what you check: [POST /users] should be 201 status code and prtoperties ...